packages/next/src/index.ts

```diff
@@ -1094,6 +1094,7 @@ export const build: BuildV2 = async ({
           shouldAddHelpers: false,
           shouldAddSourcemapSupport: false,
           supportsMultiPayloads: !!process.env.NEXT_PRIVATE_MULTI_PAYLOAD,
+          framework: 'nextjs',
```

Comment:
Might be helpful to have the version for additional context e.g. `nextjs@${detectedVersion}`